index.html

```diff
@@ -0,0 +1,385 @@
+<!DOCTYPE HTML>
```

Comment:
Do we really need this file in the repo? Isn't this generated by mdbook?

Reply:
That is the concern that mdbook put `.html` file in the same related `.md` file directory. That is why I moved `.md` files in `doc/` to the `<book_folder>/src` directory.
Another issue for the `README.md`, the `index.html` style will not be applied except I did fix manually the css location inside `index.html` file.

For the `index.html` I have 2 options:
- Copy README to the `<book_folder>/src` directory. So each time README changed, it should be copied to that dir.
- Keep it and manually fix the css location inside the `index.html'

**Normal page inside 'clippy_book/dir`** Note html directory.
     ![image](https://user-images.githubusercontent.com/5870208/109383148-3cf1e480-78ed-11eb-853d-b359c9e30f60.png)

**Home page inside main directory** Note html directory.
![image](https://user-images.githubusercontent.com/5870208/109383173-627eee00-78ed-11eb-8f6d-b4c894cebf32.png)

Reply:
I **really** don't think, that you have to add any css or html files to get the standard formatting of the book, which we want. If I'm mistaken, please link me to the documentation explaining why a seems-to-be-generated html file is necessary in the repo.

Reply:
Generating css and html done automatically by `mdbook build`. So I did `mdbook clean` so the extra (html, css, js) files removed and only main files kept.